css/theme.scss

```diff
@@ -19,7 +19,7 @@
     --hero-button-block-background: var(--lighter-gray);
 
     --hero-button-primary-border-color: var(--color-primary);
-    --hero-buttom-primary-background: var(--color-primary);
+    --hero-button-primary-background: var(--color-primary);
```

Comment:
ah nice catch, but that value is also wrong in the original theme, I'll go and fix it there as well